experimental/dds/tree/src/SharedTree.ts

```diff
@@ -1401,7 +1400,7 @@ export class SharedTree extends SharedObject<ISharedTreeEvents> implements NodeI
 	}
 
 	private serializeEdit<TChange>(preparedEdit: Edit<TChange>): Edit<TChange> {
```

Comment:
@CraigMacomber, @noencke - FYI the "semi-serializable" state is going away.  Handles will remain unencoded all throughout the DDS implementation.  `SharedObject.submitLocalMessage` is becoming the chokepoint for making the handles serializable.  And eventually that will move up to the ContainerRuntime.

I think we should probably open a small task to go clean up all the places that use this callback, etc.

Reply:
@connorskees - Until then, let's inline this (i.e. use `(x) => x`), or at least rename this private function.